test/scripts/filters/external_link.js

```diff
@@ -26,13 +26,23 @@ describe('External link', () => {
       '# External link test',
       '1. External link',
       '<a href="https://hexo.io/">Hexo</a>',
-      '2. Internal link',
+      '2. External link with "rel" Attribute',
+      '<a rel="external" href="https://hexo.io/">Hexo</a>',
+      '<a href="https://hexo.io/" rel="external">Hexo</a>',
+      '<a rel="noopenner" href="https://hexo.io/">Hexo</a>',
+      '<a href="https://hexo.io/" rel="noopenner">Hexo</a>',
+      '<a rel="external noopenner" href="https://hexo.io/">Hexo</a>',
+      '<a href="https://hexo.io/" rel="external noopenner">Hexo</a>',
+      '3. External link with Other Attributes',
+      '<a class="img" href="https://hexo.io/">Hexo</a>',
+      '<a href="https://hexo.io/" class="img">Hexo</a>',
+      '4. Internal link',
       '<a href="/archives/foo.html">Link</a>',
-      '3. Ignore links have "target" attribute',
+      '5. Ignore links have "target" attribute',
```

Comment:
My preference is to have "noopener" when target is "_blank", and ignore links with target value other than "_blank".

Maybe this is a way to disable "rel=noopener" specifically without disabling "config.external_link"?

Reply:
It can disable `rel=noopener but I don't know if it is designed to do that. Since this PR only dropped the cheerio, I just keep that behavior.

Reply:
Surprise feature huh 🤣 . I think it's ok to have keep it.

If the behavior is undesired, user can always remove `target="_blank"` and enable `external_link`.